### ISSUE TITLE
s3api PostPolicy need detect and set Content-Type

### ISSUE THREAD

**Issue by versionkill** [seaweedfs/seaweedfs#4461]
![image](https://github.com/seaweedfs/seaweedfs/assets/19726038/e668c6c3-0dd7-4bd1-b60a-6fa54d7c8a3b)

**Describe the bug**
Currently, s3api PostPolicy request header was `multipart/form-data; boundary=--------------------------xxxxx`，causing the stored file Content Type to be the same, which is incorrect. Please detect and set file Content Type, at least it should be octets/stream.

 Additionally, it is recommended that the successful upload of POSTPolicy returns a code of 204, not 200.

**Comment by chrislusf** [comment]
not clear how to reproduce this.

**Comment by versionkill** [comment]
![image](https://github.com/seaweedfs/seaweedfs/assets/19726038/5494c20c-8af3-4836-921d-629d33bd0927)
![image](https://github.com/seaweedfs/seaweedfs/assets/19726038/e1cc3e85-c3c2-4540-b0bd-f19732cc0f60)
![image](https://github.com/seaweedfs/seaweedfs/assets/19726038/bf886bd7-51d4-4765-882a-093528b5d3ad)

**Comment by ibmkhd** [comment]
Hi @versionkill did you solve this, as am facing the same issue when uploading using "s3 presigned post"  URL?

**Comment by namlh-eureka** [comment]
yes, i'm facing same issue. the "s3 presigned post" URl when upload should return 204 by default, instead of 200.

https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectPOST.html
